partiql-parser/src/prelude.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates.
+
+//! Convenience export of common traits and basic types that are almost always
+//! needed when using the parser APIs.
+
+pub use crate::result::ParserError;
+pub use crate::result::ParserResult;
+pub use crate::result::Position;
```

Comment:
The idea here is we'll put our most basic common types/traits that anyone using the parser will invariably need to import so users can `use partiql_parser::prelude::*`.  I've seen this pattern in various libraries, but it isn't totally ubiquitous so perhaps we'll back out of this later...